gradle.properties

```diff
@@ -7,14 +7,14 @@ artifactory_contextUrl=https://labkey.jfrog.io/artifactory
 sourceCompatibility=17
 targetCompatibility=17
 
-gradlePluginsVersion=1.36.1
+gradlePluginsVersion=1.38.0
```

Comment:
We're going to need a new gradle plugin release for this change. We have a method there called `getRemoteApiProjectPath` that is used for declaring dependencies in various places